examples/tutorials/Part 1 - The Basic Tools of Private Deep Learning.ipynb

```diff
@@ -83,7 +83,11 @@
    "source": [
     "# Run this cell to see if things work\n",
     "import syft as sy\n",
+<<<<<<< HEAD
```

Comment:
Please remove these lines. The indicate where there was a merge conflict

Reply:
On second thought this file should not be in this pr.
Please run `git checkout dev -- 'examples/tutorials/Part 1 - The Basic Tools of Private Deep Learning.ipynb'`
Then commit the changes